### ISSUE TITLE
Multiple images upload and reference

### ISSUE THREAD

**Issue by brhahlen** [TandoorRecipes/recipes#2226]
### Is your feature request related to a problem? Please describe.

Recently started using Tandoor and am quite excited about it. Looks really good, and works well.

The only thing that is a bit "cumbersome" is to add multiple images into a recipe.
I am currently doing a cold smoke of a salmon, and want to add multiple images per step, to ensure that I have the visual reference later.
However, to add them now, I have to upload them myself to the `mediafiles/files/<recipe_dir>` folder, and then link them via Markdown on the HTTP link.

### Describe the solution you'd like

It would be awesome to have an interface to upload multiple files, keeping the name as you are uploading with, so that you can easily reference the files from within the recipe.

So basically, suppose I have "Meatballs" as a recipe, and 6 images to use in the recipe.
I would then want to upload all 6 images, names `meatball-1.jpg`, `meatball-2.jpg`, etc, and then be able to reference them in markdown, ideally as `![](files/meatball/meatball-1.jpg)`, so you don't need to reference the full HTTP address.

**Comment by vabene1111** [comment]
currently images are best added using file steps, manually uploadng to mediafiles works but is not really supported/meant to be this way. 

In the future this will be improved (already on the road map/in some other issues)

**Comment by brhahlen** [comment]
Cool! Thanks for the reply!
And apologies for my belated response, got lost in the Holiday business.

Add image using the file steps only allows you to add one image per step though, which in most cases is fine, but in this particular case that I had, it did not :)

Looking forward to seeing the improvements, and thanks again for your hard work!
Wishing you a very good 2023!
